desdeo_problem/testproblems/RiverPollution.py

```diff
@@ -21,8 +21,9 @@ def river_pollution_problem(five_obj: bool = True, var_iv: np.array = ([0.5, 0.5
         MOProblem: a problem object.
     """
     
-    if 1 <= var_iv[0] or var_iv[1] <= 0.3:
-        raise ValueError("Initial variable values need to be between lower and upper bounds")
+    for x in var_iv:
```

Comment:
Same thing here as in the car crash problem. Check the arrays using casting instead of loops.